projects/systelab-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "systelab-components",
-  "version": "16.0.3",
```

Comment:
For reference, last tag was 16.1.0 despite package.json version was not updated, that's why I set it as 16.1.1 (and not 16.0.4)